snuba/query/processors/physical/column_filter_processor.py

```diff
@@ -0,0 +1,13 @@
+from typing import Sequence
+
+from snuba.clickhouse.query import Query
+from snuba.query.processors.physical import ClickhouseQueryProcessor
+from snuba.query.query_settings import QuerySettings
+
+
+class ColumnFilterProcessor(ClickhouseQueryProcessor):
+    def __init__(self, column_filters: Sequence[str]) -> None:
+        pass
```

Comment:
probably want to at least set the column_filters 
`self.__column_filters = column_filters`